naurffxiv/src/pages/ultimates/_meta.js

```diff
@@ -1,5 +1,6 @@
 export default {
   fru: {
+    display: 'hidden'
```

Comment:
the latest build still has it shown. not sure if that's intended

Reply:
This only hides it from the sidebar (as claire previously requested)
I'll remove it from the dropdown